bandit/options.h

```diff
@@ -93,6 +93,9 @@ namespace bandit {
                 {DRY_RUN, 0, "", "dry-run", argument::None,
                     "  --dry-run, "
                     "\tSkip all tests. Use to list available tests"},
+                {REPORT_TIMING, 0, "", "report-timing", argument::None,
+                    "  --report-timing, "
+                    "\tAdd timing information to info reporter"},
```

Comment:
This should probably be something like **"Instruct reporter to report timing information"** (assuming that not every reporter will handle it).

Reply:
Ok changed.